swedish/titta-ut/macka/macka.md

```diff
@@ -1,3 +1,9 @@
 Du går till köket för att göra dig en macka. Väl där hittar du 
 samtliga ingredienser som behövs och äter din nyligen förberedda 
-delikatess i lugn och ro.
\ No newline at end of file
+delikatess i lugn och ro.
+
+När mackan väl landat i magen känner du ett enormt sötsug. På bordet framför dig ligger en marshmallow. Vad gör du?
+
+[Äter en marshmallow.](marshmallowen/marshmallowen.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.